security/pkg/server/ca/authenticate/kubeauth/kube_jwt.go

```diff
@@ -61,14 +63,22 @@ func NewKubeJWTAuthenticator(
 	meshHolder mesh.Holder,
 	client kubernetes.Interface,
 	clusterID cluster.ID,
+	clusterAliases map[string]string,
```

Comment:
I think we could leave this as cluster.ID instead of string and remove the below conversion. Let me know if I am missing something!
```suggestion
	clusterAliases map[cluster.ID]cluster.ID,
```

Reply:
@jaellio So we need to handle the `map[string]string` (representation received from CMD args) to `map[cluster.ID]cluster.ID` somewhere.

I copied such approach from https://github.com/istio/istio/blob/f392af63aa0ca33329c2d5d64d988fcb596719d7/pilot/pkg/xds/discovery.go#L164-L167 

What do you think, should we handle this type conversion in `bootstrap/server.go` and pass it as `map[cluster.ID]cluster.ID` to both XDS and KubeJWT constructors?

Reply:
Tried to do so in https://github.com/istio/istio/commit/96fe8cb4195bb73e71bd8e6906e962bfc8193aee

Reply:
Ah I was looking at the ClusterAlias definition in [discovery.go]( https://github.com/dm3ch/istio/blob/a02a23939a35fa735edd1a097dc9810a024300b5/pilot/pkg/xds/discovery.go#L129) rather than [controller.go]( https://github.com/dm3ch/istio/blob/a02a23939a35fa735edd1a097dc9810a024300b5/pilot/pkg/serviceregistry/kube/controller/controller.go#L117). I think you're good to leave it the way you had it before.